pkg/assembler/graphql/schema/certifyVEXStatement.graphql

```diff
@@ -67,6 +69,13 @@ enum VexJustification {
   VULNERABLE_CODE_CANNOT_BE_CONTROLLED_BY_ADVERSARY
   INLINE_MITIGATIONS_ALREADY_EXIST
   NOT_PROVIDED
+  REQUIRES_CONFIGURATION
```

Comment:
Same here, can use append to `StatusNotes` and map to existing for the justifications also

Reply:
so here it would be status + justification in the `StatusNotes` field